clang/include/clang/AST/Type.h

```diff
@@ -3602,6 +3602,10 @@ class MemberPointerType : public Type, public llvm::FoldingSetNode {
   }
 
   NestedNameSpecifier *getQualifier() const { return Qualifier; }
+  CXXRecordDecl *getCXXRecordDecl() const;
```

Comment:
Could you make this private for now to give some space for @mizvekov to have a discussion for the proper name of the getter, while still being able to land this?

Reply:
Sure, done. @mizvekov I'll also create a follow-up PR to rename the method - we can discuss the most appropriate choice there.